### PR TITLE
Add secure works integration

### PR DESCRIPTION
**Pull request by Spivaka**
fixes https://github.com/demisto/server/issues/7627

### CODE REVIEW

File: Integrations/integration-SecureWorks.yml [hunk 1]
```diff
@@ -0,0 +1,485 @@
+commonfields:
+  id: SecureWorks
+  version: -1
+name: SecureWorks
+display: SecureWorks
+category: Utilities
```
Comment (aviadl) [inline Integrations/integration-SecureWorks.yml]: Is this the correct category ?

File: Integrations/integration-SecureWorks.yml [hunk 2]
```diff
@@ -0,0 +1,485 @@
...
...VgKWLQmKVy5qGbBor61SOtb6k2ukMCyGDMP8PyqEIIQwGvXD+uUt4KGuKdbpvRppohnLaTWDo3ELPH7JR+lAET9s27br3M1Xd10xv7HpN8WM8McUOyXfDioROX74JkG/C6a0JPzuiX1QP0R5ZiVwuhzw9PM74YrVHfC9W94LR4/FIRCmLultdqoRvpJWpvKLZfCfYSNuOp2B6uP6PApUsoaiNSGl/dlEseebvzj6oUcPZRY9un2AMu0B6qLQ+cbzzWvf3LDj/T0ND9ZHqr9EyTZJXYeE0wltvgJ+NNFRmIp9ll0O0fDVroJG/5PLImy0zCHHLIYMw7y+PMJMrgjVYQ9ceE4zUHdFysc9U8S/8LgkvVAyKXs/BE45BmGXG8aTJddt923/9b2fa7rFFQh9v6Slx6kSz2TBIapHocMoaoJSQAyF4TfUhKC+2i+q69C/q8M+CAU8QjR96M15PeZM+Tpr5r3tJC5zkWxRyxd5kL4b9LqhttqFguhtNPL6nId3JzY8sWvfjTtGJhof2lsE80hCNBqv7qyBSz68rvy2C7onVi3puBlP6z6v6MSiQ1xzCK9P7JNSsXK7A31ZxMJWkpJFII8kidrM9KXOQ4fFkGGY14c3SGXAElkdqtG7W728Tsz7U8bZCz+kqfKTL9De3BiB47sLkIlWiW7q7jof3Pf7vXClZPzTD26+/JpaT2h1Gh26RB7AWf7jtURmevHBGT0/VYZKp5OABS55ElwKbVE6TNDn5BPDa+58trDu8IR15baDfYFD/ehyHs+LthfqbCe88U218MZzlm67at2iF+bWRb9Dzm22DPkBg7p9aBAo24n704kRJ6UdWPDyr040dmNYDBmGeV1AEaNUBq+zzgmr5gfBLWuQLZoikOWsolU2wevy77t4TTs8/eQ2KNbOBYkqCVF1mZYu+M3DfXDuoR8t++TVy/a/Zf2C27rC/sedAP2UAlFCz29ctj3BMwkeiaFqd0ADpxO9ycrPam2PzVEC6D6YkuYPj4717BxIR/r6By7fezDbcGwiJQ3FUHqp2zG1W2uogqUXtsKbl9ZPbljZ+F8NLXU/rQqoz4saaCD6SYpOJ9QMQ+ZhwGLIMMzfNlSLs6XKBRctDoleWAb+2/9HtsFky5RMS9m38fKer/1qy85P7dq9C1ztXaCUDQhRzdTGLhiYzMHHv/Zc27f/+8Ddaxc3
```
Comment (aviadl) [inline Integrations/integration-SecureWorks.yml]: Failed to - > Failed on request

Comment (aviadl) [inline Integrations/integration-SecureWorks.yml]: You dont need this line AFAIK

Comment (Spivaka) [inline Integrations/integration-SecureWorks.yml]: This comment is scrambled. don't have a line here... which line do you mean?

Comment (aviadl) [inline Integrations/integration-SecureWorks.yml]: line 215

**Pull request merged**